.readthedocs.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+formats: all
+sphinx:
+  configuration: docs/conf.py
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"  # Keep in sync with .github/workflows/tests.yml
```

Comment:
It may make sense to put a similar message in https://github.com/zytedata/zyte-parsers/blob/bfb2a59e868728f124d1a175d1ff3cff4c1f2b17/.github/workflows/test.yml#L47